lib/Command/GetRecommendations.php

```diff
@@ -72,7 +72,7 @@ public function execute(InputInterface $input, OutputInterface $output) {
 			return 1;
 		}
 
-		if ($input->hasArgument('max')) {
+		if ($input->getArgument('max')) {
```

Comment:
Is it not there anymore?

Reply:
it is, but `hasArgument` always return `true` for a defined argument :shrug: